laravel.md

```diff
@@ -0,0 +1,45 @@
+# Laravel
+
+**Prerequisite knowledge:** php, oop, mysql, html, frameworks, mvc
+
+## Intro
+Laravel is a free, open-source PHP web framework, created by Taylor Otwell and intended for the development of web applications following the model–view–controller (MVC) architectural pattern and based on Symfony. Some of the features of Laravel are a modular packaging system with a dedicated dependency manager, different ways for accessing relational databases, utilities that aid in application deployment and maintenance, and its orientation toward syntactic sugar. [[Source](https://en.wikipedia.org/wiki/Laravel)]
+
+## Classification / acceptance questions
+
+### Apprentice aka _My First TODO app_
+* I know what Laravel is
+* I can use Laravel router to interact with the end user
+* I can create simple blade templates to separate presentation layer from business logic
+* I can Create, Read, Update, Delete entries in DB through Eloquent Model 
+* I can use MVC design pattern in Laravel
+
+### Intermediate aka _Beta users can use my TODO app_
+**Prerequisite knowledge:** cookies, session, authorization
+* I can use artisian CLI for DB migration and code generation
+* I can set data in users session
+* I can introduce Authorization to my application - only logged in users can see the content
+* I can validate data provided by user to make sure the content is in the right shape
+* I can limit permissions to introduce admin role
+
+### Advanced aka _My TODO app has REST API_
+**Prerequisite knowledge:** rest, http,
+* I can read headers from request and set headers on a response
+* I can create REST API in my app
+* I can use Lavarel transformers to make the output consistent
```

Comment:
What do you think about adding integration with Vue here?
I personally have a mixed feeling about putting it in docs about the Laravel, but it's very common to follow that path.

Reply:
I'm not sure about the vue but I would mention assets somewhere for sure.

Reply:
Good Idea, I put it in `Intermediate` level